README.md

```diff
@@ -183,7 +183,7 @@ Please take careful note of the following points, before using blobfuse:
 	- to go back to your default logging level (provided in command line options) 
 		- remove the `logLevel` entry from config file 
 		- after saving config file send `SIGUSR1` to running instance of blobfuse.
-- By default logs are directed to system-configured syslog file e.g. /var/log/syslog
+- By default logs are directed to system-configured syslog file e.g. /var/log/syslog or var/log/message (Depending Upon the Linux OS Family)
```

Comment:
that's right, but change the phrase system-configured syslog file to system-configured log file

Reply:
Thanks @NaraVen - Thanks for the review. The request has been re-submitted